src/Instrumentation/Psr6/composer.json

```diff
@@ -0,0 +1,47 @@
+{
+  "name": "open-telemetry/opentelemetry-auto-psr6",
+  "description": "OpenTelemetry auto-instrumentation for PSR-6 (Common Interface for Caching Libraries).",
+  "keywords": ["opentelemetry", "otel", "open-telemetry", "tracing", "psr6", "instrumentation"],
+  "type": "library",
+  "homepage": "https://opentelemetry.io/docs/php",
+  "readme": "./README.md",
+  "license": "Apache-2.0",
+  "minimum-stability": "dev",
+  "prefer-stable": true,
+  "require": {
+    "php": "^8.0",
+    "ext-opentelemetry": "*",
+    "open-telemetry/api": "^1.0",
+    "open-telemetry/sem-conv": "^1.23",
+    "psr/cache": "^1 || ^2 || ^3"
+  },
+  "require-dev": {
+    "friendsofphp/php-cs-fixer": "^3",
+    "phan/phan": "^5.0",
+    "phpstan/phpstan-mockery": "^1.1.0",
+    "phpstan/phpstan": "^1.1",
+    "phpstan/phpstan-phpunit": "^1.0",
+    "psalm/plugin-phpunit": "^0.16",
+    "open-telemetry/sdk": "^1.0",
+    "phpunit/phpunit": "^9.5",
+    "vimeo/psalm": "^4.0"
```

Comment:
try updating this to v5 to fix the failure on 8.3

Reply:
Ok